### ISSUE TITLE
Repeated "kubectl" in the output of command "kubectl exec"

### ISSUE THREAD

**Issue by zzxwill** [kubernetes/kubectl#987]
**What happened**:
When executing `kubectl exec`, the output shows there is a repeated `kubectl` in `Use kubectl kubectl exec [POD]`.
(I did a quick query whether someone already reported it, and found no record.)

```
# kubectl exec -it afdfsfsaf bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl kubectl exec [POD] -- [COMMAND] instead.
```

**What you expected to happen**:

# kubectl exec -it afdfsfsaf bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
```

**How to reproduce it (as minimally and precisely as possible)**:
<!-- Please make sure you are able to reproduce the bug using a supported version and version skew of Kubernetes. See https://kubernetes.io/docs/setup/release/version-skew-policy for which versions and are currently supported.
-->

Execute `kubectl exec -it afdfsfsaf bash`

**Anything else we need to know?**:

**Environment**:
- Kubernetes client and server versions (use `kubectl version`):
```
# kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.3", GitCommit:"2e7996e3e2712684bc73f0dec0200d64eec7fe40", GitTreeState:"clean", BuildDate:"2020-05-20T12:52:00Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-30T20:19:45Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):

**Comment by danielhelfand** [comment]
This was addressed in kubernetes/kubernetes#89919. See #887 for similar issue.

**Comment by zzxwill** [comment]
Great, thanks. Closing it.
